gsa/src/web/pages/reports/filterdialog.js

```diff
@@ -122,12 +131,24 @@ const ReportFilterDialogComponent = ({
         onSortByChange={onSortByChange}
       />
 
+      {capabilities.mayCreate('filter') &&
+        <CreateNamedFilterGroup
+          filter={filter}
+          filterName={filterName}
+          filterNameValid={filterNameValid}
```

Comment:
I am really unsure about the filter name valid code because

* The displayed message is about missing name and the prop is about a valid name
* The filter group could check for the filter name by itself
* Saving could be disable if the input doesn't contain a name (as it was before with the powerfilter)
* Saving without a name could issue an error which is displayed in a more general error handling part like with the SaveDialog
* filterNameValid is set to true as default which isn't really the case in the initial rendering where the name is missing. So maybe using filterNameInvalid would be better